### DISCUSSION TITLE
Google october event

### DISCUSSION THREAD

**Discussion by andsofine** [LawnchairLauncher/lawnchair#4983] [Q&A] [answered]
Any updates on this?

"We are teaming up with third-party launcher developers (Smart Launcher, Niagara, Nova, etc.) to launch a survey (initially made by the Smart Launcher devs) to identify how widespread OS issues are for third-party launchers."
"We'll present the findings at a Google event in October ... "
Source: https://t.me/lawnchairci/1599

How did it go? Any details?

**Comment by validcube** [comment, marked as answer]
The forms is made by people at Smart Launcher, check their announcement any information or news regarding this.

I personally not sure if this ever been presented but if it did, they might have made an announcement (and I didn't noticed it *yet*)

**Reply by andsofine** [reply]
> The forms is made by people at Smart Launcher, check their announcement any information or news regarding this.

Thank you, I'll take a look.

> I personally not sure if this ever been presented but if it did, they might have made an announcement (and I didn't noticed it yet)

Hopefully not, maybe some other reasons.
